.projenrc.ts

```diff
@@ -218,25 +239,44 @@ project.compileTask.prependExec(
   "yarn link && cd ./test-app && yarn link functionless"
 );
 project.compileTask.env("NODE_OPTIONS", "--max-old-space-size=4096");
+project.compileTask.env("TEST_DEPLOY_TARGET", "AWS");
+// project.compileTask.env("TEST_DESTROY_STACKS", "1");
+
 project.compileTask.prependExec("ts-node ./scripts/sdk-gen.ts");
 
 project.testTask.prependExec(
   "cd ./test-app && yarn && yarn build && yarn synth --quiet"
 );
-project.testTask.prependExec("./scripts/localstack");
-project.testTask.exec("localstack stop");
+// project.testTask.prependExec("./scripts/localstack");
+// project.testTask.exec("localstack stop");
 
 project.testTask.env("NODE_OPTIONS", "--max-old-space-size=4096");
-project.testTask.env("DEFAULT_REGION", "ap-northeast-1");
-project.testTask.env("AWS_ACCOUNT_ID", "000000000000");
-project.testTask.env("AWS_ACCESS_KEY_ID", "test");
-project.testTask.env("AWS_SECRET_ACCESS_KEY", "test");
```

Comment:
What were these doing before?

Reply:
setting the env creds to hit localstack.